docs/modules/usage/troubleshooting/troubleshooting.md

```diff
@@ -1,143 +1,38 @@
-    return self._request(
-           ^^^^^^^^^^^^^^
-  File "/app/.venv/lib/python3.12/site-packages/openai/_base_client.py", line 1012, in _request
-    raise self._make_status_error_from_response(err.response) from None
-openai.NotFoundError: Error code: 404 - {'error': {'code': '404', 'message': 'Resource not found'}}
+When running OpenHands, the following error is seen:
 ```
-
-**Details**
-
-This happens when LiteLLM (our library for connecting to different LLM providers) can't find
-the API endpoint you're trying to connect to. Most often this happens for Azure or ollama users.
-
-**Workarounds**
-
-* Check that you've set `LLM_BASE_URL` properly
-* Check that the model is set properly, based on the [LiteLLM docs](https://docs.litellm.ai/docs/providers)
-  * If you're running inside the UI, be sure to set the `model` in the settings modal
-  * If you're running headless (via main.py) be sure to set `LLM_MODEL` in your env/config
-* Make sure you've followed any special instructions for your LLM provider
-  * [Azure](/modules/usage/llms/azure-llms)
-  * [Google](/modules/usage/llms/google-llms)
-* Make sure your API key is correct
-* See if you can connect to the LLM using `curl`
-* Try [connecting via LiteLLM directly](https://github.com/BerriAI/litellm) to test your setup
-
----
-### `make build` getting stuck on package installations
-
-**Symptoms**
-
-Package installation stuck on `Pending...` without any error message:
-
-```bash
-Package operations: 286 installs, 0 updates, 0 removals
-
-  - Installing certifi (2024.2.2): Pending...
-  - Installing h11 (0.14.0): Pending...
-  - Installing idna (3.7): Pending...
-  - Installing sniffio (1.3.1): Pending...
-  - Installing typing-extensions (4.11.0): Pending...
+Launch docker client failed. Please make sure you have installed docker and started docker desktop/daemon.
 ```
 
-**Details**
-
-In rare cases, `make build` can seemingly get stuck on package installations
-without any error message.
-
-**Workarounds**
-
-The package installer Poetry may miss a configuration setting for where credentials are to be looked up (keyring).
-
-First check with `env` if a value for `PYTHON_KEYRING_BACKEND` exists.
-If not, run the below command to set it to a known value and retry the build:
-
-```bash
-export PYTHON_KEYRING_BACKEND=keyring.backends.null.Keyring
-```
+**Resolution**
+1. Confirm `docker` is running on your system. You should be able to run `docker ps` in the terminal successfully.
+2. If using Docker Desktop, ensure `Settings > Advanced > Allow the default Docker socket to be used` is enabled.
+3. Depending on your configuration you may need `Settings > Resources > Network > Enable host networking` enabled in Docker Desktop.
+4. Try reinstalling Docker Desktop.
 
----
 ### Sessions are not restored
 
-**Symptoms**
-
-OpenHands usually asks whether to resume or start a new session when opening the UI.
-But clicking "Resume" still starts a fresh new chat.
-
-**Details**
+**Description**
```

Comment:
Kept this Sessions are not restored here. But I feel like troubleshooting is not a good place for it.
Is there a better place to move it to?

Reply:
IIRC Robert was suggesting an Advanced Configuration section, and linked from the main README. That could work?

Reply:
Alright. Yeah just remembered the thread. I'll take a stab at it.